### PR TITLE
Switch animated snapshot format from GIF to APNG

### LINKED ISSUE #10: Animated GIF snapshots show incorrect colors
I've found that some of the snapshot GIFs that I've generated for real animations (not the simplified test animations in the framework) don't look correct, going so far as having labels be the wrong color during some frames and in one instance a single frame of the animation being solid gray. It looks like other people have seen [similar issues](https://stackoverflow.com/questions/44960714/ios-colors-incorrect-when-saving-animated-gif) with large GIFs, so it appears that CoreGraphics has some problems with quantizing colors in large images.

### PR DESCRIPTION
**Pull request by NickEntin**
Resolves #10

### CODE REVIEW

**Comment by NickEntin** [comment]
Note that this is a breaking change since previously recorded snapshots will no longer pass. I considered making this support both GIF and APNG, but ultimately I don't think GIF turned out to be a good representation of the animations, so I think moving away from it completely is the right move.

**Review by efirestone — APPROVED** [review]

**Comment by sethfri** [comment]
belated 👍

File: Stagehand/Classes/Testing/FBSnapshotTestCase+AnimationGIF.swift [hunk 1]
```diff
@@ -505,30 +505,32 @@ extension FBSnapshotTestCase {
...
-                kCGImagePropertyGIFUnclampedDelayTime: lastFrameDuration,
+            kCGImagePropertyPNGDictionary: [
+                kCGImagePropertyAPNGUnclampedDelayTime: lastFrameDuration,
             ],
         ] as CFDictionary
 
         let url = URL(fileURLWithPath: NSTemporaryDirectory())
             .appendingPathComponent(UUID().uuidString)
-            .appendingPathExtension(".gif")
+            .appendingPathExtension("png")
```
Comment (sethfri) [inline Stagehand/Classes/Testing/FBSnapshotTestCase+AnimationGIF.swift:521]: Huh, I guess the `.` in `.gif` was just getting ignored before?

File: Example/Unit Tests/AnimationSnapshotTests.swift [hunk 2]
```diff
@@ -172,6 +172,18 @@ final class AnimationSnapshotTests: FBSnapshotTestCase {
         SnapshotVerify(animation: animation, on: element, using: view)
     }
 
+    func testAnimationWithPartialTransparency() {
```
Comment (sethfri) [inline Example/Unit Tests/AnimationSnapshotTests.swift:175]: nice add

File: Stagehand/Classes/Testing/FBSnapshotTestCase+AnimationGIF.swift (hunk 1 above)
Comment (NickEntin) [reply Stagehand/Classes/Testing/FBSnapshotTestCase+AnimationGIF.swift:521]: It was actually outputting a file that ended in `..gif`, which still works but is unnecessary.

**Pull request merged**